src/ui/components/CreateIdentifier/components/IdentifierStage4.tsx

```diff
@@ -81,8 +81,9 @@ const IdentifierStage4 = ({
   };
 
   useEffect(() => {
+    const otherIdentifierContacts = [...state.selectedConnections];
     setOtherIdentifierContacts(
-      state.selectedConnections.sort(function (a, b) {
+      otherIdentifierContacts.sort(function (a, b) {
```

Comment:
Do we need the changes to this file? Because this is a clone of the array which doesn't seem necessary

Reply:
Yes, this is necessary because `state.selectedConnections` is a read-only as in the picture I showed you on Slack yesterday. Without this change, React will throw the error.

Reply:
Got it! No problem. React always enforces all these inefficient clones ;)